tests/chainer_tests/training_tests/triggers_tests/test_minmax_trigger.py

```diff
@@ -26,6 +26,10 @@ def epoch(self):
         return 1
 
     @property
+    def epoch_detail(self):
```

Comment:
This is because `MinMaxTrigger` calls `IntervalTrigger` internally.
https://github.com/pfnet/chainer/blob/master/chainer/training/triggers/minmax_value_trigger.py#L24
https://github.com/pfnet/chainer/blob/master/chainer/training/util.py#L39

Without this modification, the tests of `MinMaxTrigger` failled.